R/calcHistorical.R

```diff
@@ -325,9 +325,14 @@ calcHistorical <- function() {
     as.magpie(spatial = 4, temporal = 1, data = ncol(.data))
 
   # Steel Stock ----
-  steelStock <- calcOutput("SteelStock", aggregate = F)
+  steelStock <- calcOutput("SteelStock", aggregate = FALSE)
   steelStock <- add_dimension(steelStock, dim = 3.1, add = "model", nm = "Mueller")
 
+  # industry value added ----
+  industry_VA <- calcOutput(type = 'UNIDO', subtype = 'INDSTAT2',
```

Comment:
Please no longer add new functionality to calcHistorical, but call the calc function directly in fullVALIDATIONREMIND. 

(`calcHistorical` is legacy code and will be split up in the long run. Creating an input data revision is done using the retrieveData wrapper calling `full...` functions)